packages/vue-pdf-viewer/src/style/components/_pdf_viewer.scss

```diff
@@ -77,6 +77,49 @@
             }
         }
     }
+
+    .controls-new {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        position: absolute;
+        z-index: 100;
+        top: 16px;
+        padding: 0 24px;
+        width: 100%;
+
+        .pages {
+            padding: 8px;
+            background: #f1f5f9;
+            color: rgba(#14213d, 0.8);
```

Comment:
We could use local css variables for colors, so we can at least reuse them in here. A prefix like `--pdfviewer-color-...` would be good to ensure that we don't override any other css variables defined further up.

Reply:
Alright, added a new variable for it